<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>458</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>458</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Automation Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="keyActionButton">
     <property name="text">
      <string>Key Press</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="loopedBufferButton">
     <property name="text">
      <string>Looped Buffer</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2" rowspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>77</width>
       <height>164</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pasteBufferButton">
     <property name="text">
      <string>Paste Buffer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="upItemButton">
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="downItemButton">
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="dublicateButton">
     <property name="text">
      <string>Dublicate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="openAppButton">
     <property name="text">
      <string>Open App</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="actionsLabel">
     <property name="text">
      <string>Actions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="currentLoopsLabel">
     <property name="text">
      <string>Loops Count: 0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="15">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="clickActionButton">
     <property name="text">
      <string>Mouse Click</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="editButton">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="delayButton">
     <property name="text">
      <string>Delay</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
